demo/admin/src/products/future/ProductsGridInitialFitlerWithIsAnyOf.cometGen.ts

```diff
@@ -0,0 +1,52 @@
+import { type future_GridConfig as GridConfig } from "@comet/cms-admin";
+import { type StaticSelectLabelCellContent } from "@comet/cms-admin/lib/generator/future/generator";
+import { type GQLProduct, type GQLProductStatus } from "@src/graphql.generated";
+
+/**
+ * RequiredMap is a map, that forces all string literals of Generic K to be defined
+ */
+export type RequiredMap<K extends string, T> = { [Keys in K]: T };
```

Comment:
`Record<GQLProductStatus, StaticSelectValueOption>` does the trick as well

Reply:
true 👍